src/Raven.Server/Documents/Sharding/ShardedDocumentsStorage.cs

```diff
@@ -54,7 +54,24 @@ protected override DocumentPutAction CreateDocumentPutAction()
     {
         return new ShardedDocumentPutAction(this, _documentDatabase);
     }
-    
+
+    protected override void SetDocumentsStorageSchemas()
+    {
+        DocsSchema = Schemas.Documents.ShardingDocsSchemaBase;
+        TombstonesSchema = Schemas.Tombstones.ShardingTombstonesSchema;
+        CompressedDocsSchema = Schemas.Documents.ShardingCompressedDocsSchemaBase;
+    }
+
+    protected override long ReadLastAttachmentsEtagInternal(Transaction tx)
+    {
+        var table = tx.OpenTable(ShardingAttachmentsSchemaBase, AttachmentsMetadataSlice);
```

Comment:
sth is off here, you are reading inside DocumentsStorage the AttachmensEtag? shouldn't we use the AttachmentStorage?

Reply:
I see you removed that from the AttachmentsStorage, not sure why?

Reply:
we actually read all storages (counter, ts, etc) last etags in `DocumentsStorage`,

Reply:
done